frontend/src/components/Comments/CommentCard.vue

```diff
@@ -20,18 +21,19 @@ export default {
     id: String,
     code: String,
     comment: Object,
-    authenticated: Boolean
+    authenticated: Boolean,
+    meta: {
```

Comment:
the meta field is actually in the comment object above!

Reply:
so, this meta field is always undefined

Reply:
Ahh I see.